src/OpenTelemetry/Trace/NoopSpanBuilder.cs

```diff
@@ -50,14 +50,14 @@ public override ISpanBuilder SetRecordEvents(bool recordEvents)
             return this;
         }
 
-        internal static ISpanBuilder CreateWithParent(string spanName, SpanKind kind, ISpan parent = null)
+        internal static ISpanBuilder SetParent(string name, SpanKind kind, ISpan parent = null)
```

Comment:
Is this rename part of API changes?

Reply:
It doesn't seem consistent with Java

```java 
Span span = tracer.spanBuilder("MyRequest")
          .setParent(spanContext)
          .setSpanKind(Span.Kind.SERVER).startSpan();
```

https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/Tracer.java#L219

Reply:
Agreed this is not consistent.  My initial thought was to create a new issue for alignment with Java.  Main reason to address as a new issues was to determine whether to follow Java exactly or follow a fluent builder pattern instead.

Reply:
I'd suggest to concentrate on the removing hasRemoteParent and align Tracer operations later